_tags

```diff
@@ -4,7 +4,7 @@ true: warn(@5@8@10@11@12@14@23-24@26@29@40), bin_annot, safe_string, debug
 <node_modules/**>: -traverse
 "formatTest": -traverse
 "src": include
-<src/*>: package(menhirLib unix compiler-libs.common ocamlbuild findlib easy-format BetterErrors merlin_extend re.str)
+<src/*>: package(menhirLib unix compiler-libs.common ocamlbuild findlib easy-format BetterErrors merlin_extend re.str ppx_deriving.std yojson ppx_deriving_yojson)
```

Comment:
I've always wondered what flags *actually* get passed to the compiler. I want to know so that we can support ppx in jengaboot (instead of ocamlbuild `_tags` which are super annoying).